tools/attribution/attribution.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Generates attributions for dependencies of Twoliter
+# Meant to be run from Bottlerocket's SDK container:
+# https://github.com/bottlerocket-os/bottlerocket-sdk
+
+# See the "attribution" target in the project Makefile.
+
+set -eo pipefail
+
+LICENSEDIR=/tmp/twoliter-attributions
+
+# Use the toolchain installed via `Dockerfile.attribution`
+export HOME="/home/attribution-creator"
+source ~/.cargo/env
+
+# Source code is mounted to /src
+# rustup will automatically use the toolchain in rust-toolchain.toml
+cd /src
+
+# =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=
+echo "Clarifying crate dependency licenses..."
+/usr/libexec/tools/bottlerocket-license-scan \
+    --clarify /src/clarify.toml \
+    --spdx-data /usr/libexec/tools/spdx-data \
+    --out-dir ${LICENSEDIR}/vendor \
+    cargo --locked Cargo.toml
+
+
+# =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=  =^.^=
+echo "Clarifying cargo-cross & dependency licenses..."
+git clone https://github.com/cross-rs/cross/ /tmp/cargo-cross
+pushd /tmp/cargo-cross
+git reset --hard 7b79041
```

Comment:
Where do these two Git commit hashes come from? Can we pass them in as variables or in a config file?

Reply:
These commit hashes are what we lock to in the GitHub Actions workflow for performing a release. Per your other comment, I noted that we aren't distributing these tools with Twoliter and have just removed them from the attributions script!